packages/tradershub/src/routes/TradersHubRoute/TradersHubRoute.tsx

```diff
@@ -26,11 +25,11 @@ const TradersHubRoute = () => {
     const isSwitcherVisible = isDIEL && isReal;
     const isTotalAssetsVisible = hasActiveDerivAccount || isDemo;
 
-    if (isMobile)
+    if (!isDesktop)
         return (
-            <div className='pt-16'>
+            <div className='p-20'>
```

Comment:
why is it p-20 instead of pt-20 🤔

Reply:
Because mobile has padding

Reply:
I think for mobile the overall padding is 16 as I've seen in the design. 
Can we confirm on this?

Reply:
oh yeah 16, ill change it